src/index.js

```diff
@@ -57,12 +47,17 @@ const jsonFormat = printf((info) => {
                 time: info.timestamp || '',
             },
             event: parseInfo(info),
-        },
-        err: {
+        }
+    };
+
+    if (info.stack) {
+        logObject.err = {
             name: info.name,
             stack: info.stack
-        }
-    });
+        };
```

Comment:
- `code: info.code`?
- Conditional logic to only include `info.code` if it belonged to an error?
  * Probably difficult since everything is flattened. :'(

Reply:
Indeed. This cannot be done.